src/js/electron/main.ts

```diff
@@ -1,32 +1,34 @@
+import {app} from "electron"
+import log from "electron-log"
+import "regenerator-runtime/runtime"
 import {appPathSetup} from "./appPathSetup"
+import {setupAutoUpdater} from "./autoUpdater"
+import {Brim} from "./brim"
+import globalStoreMainHandler from "./ipc/globalStore/mainHandler"
+import windowsMainHandler from "./ipc/windows/mainHandler"
+import secretsMainHandler from "./ipc/secrets/mainHandler"
+import electronIsDev from "./isDev"
+import menu from "./menu"
+import {handleQuit} from "./quitter"
+
+import {handleSquirrelEvent} from "./squirrel"
 import userTasks from "./userTasks"
 
 // app path and log setup should happen before other imports.
 appPathSetup()
```

Comment:
Shoot, I think this appPathSetup() does need to be called above the other imports. Maybe your editor auto hoisted them up to the top?

Reply:
Ah yes it did, I added a plugin to organize imports and auto-remove unused, I can change. Though manual and itests didn't throw any issues 🤔